cvat-ui/src/components/common/cvat-markdown.tsx

```diff
@@ -0,0 +1,41 @@
+import Button from 'antd/lib/button';
+import React from 'react';
+import ReactMarkdown from 'react-markdown';
+import { RouteComponentProps } from 'react-router-dom';
+
+export type UseHistoryType = RouteComponentProps['history'];
+
+const RouterLinkHOC = (history?: UseHistoryType) => (
+    function (props: { children: React.ReactNode, href?: string }): JSX.Element {
+        const { href, children } = props;
```

Comment:
Why not put `useHistory` here? 
In this case, in external components we do not need to have `history`.

Reply:
Yes, that would be better. But we cant use `useHistoty`(its undefined).  The notifications rendered by antd `notification` module are outside of our `Router`. It seems thats because its created using `React.render` [docs](https://ant.design/components/notification#why-i-can-not-access-context-redux-configprovider-localeprefixclstheme-in-notification)

Reply:
okay, I see